docs/metadata/data_structures/message.rst

```diff
@@ -44,7 +44,15 @@ Messages are given as key-value maps.
 Language Support
 ----------------
 
-If a message's ``content`` value is a string, the message will use the string as its content if displayed. Otherwise, the first localised content structure with a language that matches LOOT's current language will be used as the message's content if displayed. If there are no matches, then the first structure in English will be used.
+If a message's ``content`` value is a string, the message will use the string
+as its content if displayed. Otherwise, the first localised content structure
+with a language that matches LOOT's current language will be used as the
+message's content if displayed. If there are no exact matches, LOOT will try to
+find the nearest match. For example, if LOOT's current language has locale code
+``pt_BR``, it can display localised content structures with language code
+``pt`` or locale code ``pt_PT`` if none exist with locale code ``pt_BR``. If
```

Comment:
This isn't right, if your language is `pt_BR`, LOOT will pick `pt_BR` (preferred) or `pt` but not `pt_PT`.

Reply:
@Ortham Ah, sorry, I misunderstood. I'll correct it in a bit.

Reply:
@Ortham Ok, I rewrote it to explain the system more explicitly instead of relying solely on examples too. Hopefully I didn't misunderstand again 🙃